project/src/test/java/BucketListTests.java

```diff
@@ -0,0 +1,49 @@
+import base.BaseTestForLoggedInUserWithoutRestart;
+import data.Platform;
+import io.appium.java_client.MobileElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.allure.annotations.Issue;
+import ru.yandex.qatools.allure.annotations.TestCaseId;
+import steps.BucketListSteps;
+import utils.annotations.SkipOn;
+import utils.log.LogProvider;
+import utils.runner.Assert;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.core.IsNull.notNullValue;
+
+public final class BucketListTests extends BaseTestForLoggedInUserWithoutRestart implements LogProvider {
+	private BucketListSteps bucketListSteps = getStepsComponent().bucketListSteps();
+	private List<String> createdBucketLists = new ArrayList<>();
+
+
+	@AfterClass
```

Comment:
It would be more stable to delete created items after each method if we don't use "same screen, same bucket item" for all tests